index.js

```diff
@@ -19,14 +19,20 @@ module.exports = {
     'no-unused-vars': ['off'],
     // Use typescript version of "no-unused-vars" rule, because the default rule doesn't recognize certain typescript
     // features like enumerations
-    '@typescript-eslint/no-unused-vars': ['warn', { argsIgnorePattern: '^_', varsIgnorePattern: '^_' }],
-    'no-use-before-define': ['error', { functions: false }],
+    '@typescript-eslint/no-unused-vars': ['warn', { varsIgnorePattern: '^_', args: 'none' }],
+    'no-use-before-define': ['warn'],
+
+    // Prettier takes care of max length in code and eslint warns about prettier warnings due to the rule
+    // 'prettier/prettier'. Setting max-len to 120 for code here would add some linter warnings in code where prettier
+    // does not obey to the 120 char line length, so we're "supressing" it with 1000...
+    'max-len': ['warn', { code: 1000, comments: 120 }],
 
     // Misc
     'no-empty': ['warn', { allowEmptyCatch: true }],
 
     // TS specific things
     '@typescript-eslint/ban-ts-comment': 'warn',
```

Comment:
This wasn't part of our coding conventions but didn't we talk about allowing ts-comments with the `allow-with-description` setting?
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/ban-ts-comment.md

Reply:
Yep, obviously didn't write that down but I can remember and makes sense to me.
Changed: https://github.com/Combeenation/cbn-eslint-config/pull/4/commits/43ad924c32a1ba9a7c7e476a4fa92dfe9fc72126

Reply:
Not sure but the `@ts-expect-error` could be helpful for certain tests written in TS so I also added it 🤷‍♂️